contracts/Recruitment.sol

```diff
@@ -83,6 +84,11 @@ contract Recruitment is Ownable, ReentrancyGuard {
         require(isCompanyRegistered(_address), "Company is not registered yet");
         _;
     }
+    
+    modifier checkIfNewUser(address _address) {
```

Comment:
one modifier per check i.e:

`checkIfRegisteredCandidate`
`checkIfRegisteredReferrer`
etc 

as i one will be able to apply for jobs and refer candidates.

Reply:
@Hangout2698 said me candidate can't apply for the job directly.